openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -97,8 +97,8 @@ describe('OpenAssessment.StaffAreaView', function() {
             server.staffAreaTemplate = staffAreaTemplate;
         }
         var assessmentElement = $('#openassessment').get(0);
-        var baseView = new OpenAssessment.BaseView(runtime, assessmentElement, server, {});
-        var view = new OpenAssessment.StaffAreaView(assessmentElement, server, baseView);
+        var baseView = new OpenAssessment.BaseView(runtime, assessmentElement, server, {"STAFF_ENABLED": true});
+        var view = baseView.staffAreaView;
```

Comment:
This was already being constructed as part of `BaseView(...)`, it seems silly to orphan that and immediately create another one.